packages-native/carousel/src/Carousel.tsx

```diff
@@ -0,0 +1,166 @@
+import { createElement, Fragment, useCallback, useEffect, useState } from "react";
+import { ActivityIndicator, LayoutChangeEvent, Text, View } from "react-native";
+import { CarouselProps } from "../typings/CarouselProps";
+import { CarouselStyle, defaultCarouselStyle, LayoutStyle } from "./ui/styles";
+import { default as NativeCarousel, Pagination } from "react-native-snap-carousel";
+import deepmerge from "deepmerge";
+import { ObjectItem, ValueStatus } from "mendix";
+
+export const Carousel = (props: CarouselProps<CarouselStyle>): JSX.Element => {
```

Comment:
As now you are always returning a view as container, maybe is better to use `ReactElement` instead of `JSX.Element`

Reply:
Done